src/Microsoft.Restier.Core/ApiBase.cs

```diff
@@ -67,7 +67,14 @@ protected ApiConfiguration ApiConfiguration
                         var builder = this.ConfigureApi(new ApiBuilder());
```

Comment:
No worries after all RESTier is still in its preview stage:-) In my personal opinion:

 - I have no objection to both ways: `IServiceCollection` makes API compact while `ApiBuilder` offers a clearer meaning.
 - I just noticed that all methods of `ApiBuilder` are placed in `ApiBuilderExtensions`. Since that we have full control of the implementation of `ApiBuilder`, that `ApiBuilder` doesn't implement any interface, and that methods in `ApiBuilderExtensions` do not rely on pure abstraction either, then why do we use extension methods instead of encapsulation? Just in this sense, I would prefer `IServiceCollection`.

Please correct me if my understanding is wrong. By the way, it is not "pollution", it is just a brilliant idea that doesn't come true:-)